Makefile

```diff
@@ -35,6 +35,8 @@
 #  make install
 # Enable lua debug support
 #  make LUA=1
+# Enable developer-only feature
```

Comment:
We don't need another build define, just use ```#ifndef RELEASE``` to check if it's a debug build  We *definitely* don't want to use _GLIBXX_DEBUG for anything, that's not its purpose.